### DISCUSSION TITLE
Slow to launch

### DISCUSSION THREAD

**Discussion by melange-GRX** [aonez/Keka#1167] [Q&A]
The app works perfectly, but if I have not launched it in a day, then when I open the app, there is a long pause - sometimes up to 60 seconds. If I let it go, it eventually opens and works fine. The behavior is the same when I open the app via the contextual menu, double-click on a file (Keka is the default handler), or double click on the app's icon.

I've tried being on and off the network, with and without my VPN, etc. but the issue does not seem to be network related (my first thought was it was checking for an update). 

I am using the App Store version (currently Version 1.2.56 (5023)) on macOS 12.6 (21G115).

Has anyone else seen this? Have you found a solution? Thanks for any help!

**Comment by aonez** [comment]
Hi @melange-GRX,

The App Store version of Keka does not have network capabilities so it will never connect to the internet. The web version has network capabilities to update itself (using Sparkle) but they're enabled after Keka is launched and once a week.

I remember some user some years ago reporting something similar. I will suggest that you completely remove Keka (send it to trash and empty the trash) and reinstall it again. Maybe even a restart between the deletion and the new installation may help.

This is probably caused by some Apple security (sandbox) check. Hopefully reinstalling fixes it.
